core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -897,7 +1041,8 @@ private CarbonCommonConstants() {
   /**
    * property for configuring parallelism per segment when doing an update. Increase this
    * value will avoid data screw problem for a large segment.
-   * Refer to CARBONDATA-1373 for more details.
+   * Refer to CarbonData-1373 for more details.
+   * TODO: dynamic
```

Comment:
?

Reply:
removed, this property should can be dynamic configurable